scripts/build_review_app.sh

```diff
@@ -34,29 +35,24 @@ hokusai review_app setup $NAME
 # --tag to name the image.
 # WARNING: This is likely going to take ~10 mins on your MBP.
 # Be patient and grab some baby carrots.
-hokusai registry push --force --skip-latest --verbose --tag $NAME
+hokusai registry push --force --skip-latest --overwrite --verbose --tag $NAME
```

Comment:
`--overwrite` makes sense for `update_review_app.sh` but do we expect to run this build script more than once for a particular review app?

Nit: We might want to drop the file extension or update to `.bash` as we're specifying `bash` not `sh`as the target program.

Reply:
> --overwrite makes sense for update_review_app.sh but do we expect to run this build script more than once for a particular review app?

While it's not expected, I do think this makes the script a bit more robust: E.g – for whatever reason, but a script run fails after this command, but before review app namespace creation. But allowing us to override here, a re-build is possible. Otherwise, we would have to manually delete the image in the ECR registry to proceed (without modifying the script).

> Nit: We might want to drop the file extension or update to .bash as we're specifying bash not shas the target program.

Personally I've never seen this done before for many a bash script. Gonna opt to not rename for now.